nix-repl.el

```diff
@@ -148,14 +148,16 @@ guarantees they will be grabbed in a single call."
   "Completion at point function for Nix using \"nix-repl\".
 See `completion-at-point-functions'."
   (save-excursion
-    (let ((prefix (and (derived-mode-p 'nix-repl-mode)
-                       (executable-find nix-executable)
-                       (nix--prefix-bounds))))
+    (let* ((proc (get-buffer-process (current-buffer)))
+           (prefix (and (derived-mode-p 'nix-repl-mode)
+                        proc
```

Comment:
In theory you should probably also check the liveness of the process with `process-live-p`.

Reply:
:+1: See #108.